src/assets/JSAsset.js

```diff
@@ -173,7 +174,10 @@ class JSAsset extends Asset {
 
     return {
       js: code,
-      map: this.sourceMap
+      map: this.sourceMap,
+      '@reflect': {
```

Comment:
Rather than adding these to `generated` and then removing them later, you could just add them to `asset.cacheData`. That already gets written to the cache and you should be able to access it from the packager.

Reply:
I tried `cacheData` and I didn't get it to work (`cacheData.exports` is empty). Should I manually update it? I pushed it